week-2/implement/cases.js

```diff
@@ -13,3 +13,19 @@
 
 // You will need to come up with an appropriate name for the function
 // Use the string documentation to help you find a solution
+
+
+function toUpperCamelCase(str) {
+  // Split the input string into words based on spaces
+  const words = inputString.split(' ');
```

Comment:
The code is good. The variable inputString should be replaced with str to match the function parameter.